src/main/java/net/mcreator/generator/GeneratorTemplatesList.java

```diff
@@ -52,17 +51,7 @@ public void forEachTemplate(Consumer<ListTemplate> action, @Nullable IntConsumer
 		for (int index = 0; index < listData.size(); index++) {
 			if (beforeNextListItem != null)
 				beforeNextListItem.accept(index);
-			for (GeneratorTemplate template : templates.keySet()) {
-				if (templates.get(template).get(index)) {

```

Comment:
This condition is no longer checked?

Reply:
Conditions are now processed in the generator class.
Actually, this PR moves all the generator stuff from `GeneratorTemplatesList` back to `Generator`.

Reply:
But this method is also used for by mod element code dropdown and now it will just skip conditions

Reply:
Conditions are **still processed**, just in a proper class.
https://github.com/MCreator/MCreator/blob/410ea7f0e16b903bbc4a41a032cf70e401665f54/src/main/java/net/mcreator/generator/Generator.java#L516-L517